diff --git a/node_modules/@deepgram/sdk/src/lib/helpers.ts b/node_modules/@deepgram/sdk/src/lib/helpers.ts
index 275a1fe..ca20462 100644
--- a/node_modules/@deepgram/sdk/src/lib/helpers.ts
+++ b/node_modules/@deepgram/sdk/src/lib/helpers.ts
@@ -11,17 +11,17 @@ import {
 import { Headers as CrossFetchHeaders } from "cross-fetch";
 import { Readable } from "stream";
 import merge from "deepmerge";
-import { BROWSER_AGENT, BUN_VERSION, NODE_VERSION } from "./constants";
+// import { BROWSER_AGENT, BUN_VERSION, NODE_VERSION } from "./constants";
 
 export function stripTrailingSlash(url: string): string {
   return url.replace(/\/$/, "");
 }
 
-export const isBrowser = () => BROWSER_AGENT !== "unknown";
+export const isBrowser = () => false
 
-export const isNode = () => NODE_VERSION !== "unknown";
+export const isNode = () => true
 
-export const isBun = () => BUN_VERSION !== "unknown";
+export const isBun = () => false
 
 export function applyDefaults<O, S>(options: Partial<O> = {}, subordinate: Partial<S> = {}): S {
   return merge(subordinate, options);
